astropy/modeling/fitting.py

```diff
@@ -36,12 +36,6 @@
 from astropy.utils.exceptions import AstropyUserWarning
 
 from .optimizers import DEFAULT_ACC, DEFAULT_EPS, DEFAULT_MAXITER, SLSQP, Simplex
-from .spline import (  # noqa: F401
-    SplineExactKnotsFitter,
-    SplineInterpolateFitter,
-    SplineSmoothingFitter,
-    SplineSplrepFitter,
-)
```

Comment:
Why is this being removed? This changes the modeling API and is why the tests are failing

Reply:
A good question! I'll do some digging. Might have to add #F401 to the unfixable category.

Reply:
Looks like they are missing from the `__all__` in that file (which is an oversight). Would adding them there resolve the issue?

Reply:
Worth a try, which I see you've done 👍